src/renderer/screens/platform/App.js

```diff
@@ -16,28 +18,20 @@ type Props = {
   },
 };
 
-const manifests = {
-  wyre: {
-    name: "Wyre",
-    url: new URL(`https://ledger-live-platform-apps.vercel.app/app/wyre`),
-    // $FlowFixMe
-    icon: require("../../images/platform/wyre.png").default,
-  },
-};
-
-export default function ExchangeDapp({ match }: Props) {
+export default function PlatformApp({ match }: Props) {
   const history = useHistory();
   const { platform } = match.params;
-  const manifest = manifests[platform];
+  const manifest = useAppManifest(platform);
 
-  const handleClose = useCallback(() => history.push(`/exchange`), [history]);
+  const handleClose = useCallback(() => history.push(`/platform`), [history]);
 
   if (!manifest) {
-    return null;
+    return "oops no manifest";
```

Comment:
To me, this is not a valid solution. If we want to show an error, then render an error.
If we want to ignore this case, the redirect to another screen, but not render a screen with oops!

Reply:
Yes you are right, this is a "debug" string that was not supposed to be committed XD

Reply:
to be fixed.